experimental/Schemes/singular_locus.jl

```diff
@@ -21,14 +21,15 @@ export is_smooth
 ###       for covered schemes (the workhorse is already there...).
  
 @doc Markdown.doc"""
-    singular_locus(X::AbsSpec{<:Field, <:MPolyAnyRing}) -> AbsSpec
+    singular_locus(X::Scheme) -> Scheme
 
 Return the singular locus of `X`.
 
 For computing the singular locus of the reduced scheme induced by `X`,
 please use [`singular_locus_reduced`](@ref).
 
-Currently this command is available for affine schemes and space germs.
+Currently this command is available for affine schemes and for space germs
+(in both settings requiring that these are defined over a field).
```

Comment:
We can also make that part of the signature by defining it for `X::Scheme{<:Field}`. Sorry for the late update.